src/app/article/article.component.ts

```diff
@@ -24,15 +24,36 @@ export class ArticleComponent implements OnInit, OnDestroy {
         this.route.params
             .map(params => params['slug']).subscribe((slug) => {
             this.sub = this.NS.getArticle(slug).subscribe(
-                posts => this.article = posts[0],
+                posts => {
+                    this.article = posts[0];
+                    setTimeout(() => this.checkQuoteElement());
+                },
                 error => errorMessage = <any> error);
         });
 
     }
 
+    //Check if there are qoutes in article and add qoute symbol if so.
+    checkQuoteElement(){
+        let d: any = document.getElementsByTagName("blockquote");
+        
+        for(let i=0; i < d.length; i++){
+            //Create quote symbol element
+            let e: any = document.createElement('i');
+            let classAtt: any = document.createAttribute("class");
+            classAtt.value = "fa fa-quote-right";
+            e.setAttributeNode(classAtt);
+            let ariaAtt: any = document.createAttribute("aria-hidden");
+            ariaAtt.value = "true";
+            e.setAttributeNode(ariaAtt);
+            d[i].insertBefore(e, d[i].firstChild);
+        }
+    }
+
     ngOnInit() {
         this.searchService.activate(Archive.article);
         this.initializeData();
+
     }
 
     ngOnDestroy() {
```

Comment:
Remember to deactivate archive on destroy. Other routes may not be using the archive service